go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/lvl484/user-manager
 
 go 1.13
+
+require github.com/gorilla/mux v1.6.2
```

Comment:
Do you have dependency to `github.com/gorilla/mux`?

Reply:
We don't need this dependency
but docker container didn't start without any dependency in  go.mod

Reply:
Insert comment in go.mod

Reply:
@VasylKosiv, what was the reason of failure?

Reply:
Previously, the container fell down on this command: COPY --from=modules /go/pkg/ /go/pkg/ from Docker file. When this depdendency was added, container started running without error.
But now, I'm trying to ran the docker-compose without this dependecy, to see type of error, and container is up without error. So I don't have reason to leve this dependency

Reply:
So add unused dependency is not a solution. If my understanding is right the problem was in non existing folder with dependencies when no dependencies are specified. So you'd better to solve that issue.

Reply:
I agree whith you. But is it so important in our case? Because our project will have some dependencies for sure.

Reply:
Maybe even this dependency

Reply:
It's important to be precise, at least it's enough to run `mkdir` to make a directory.

Reply:
Thank you for advice. Fix it.